tsdb/chunkenc/float_histogram.go

```diff
@@ -44,6 +44,29 @@ func NewFloatHistogramChunk() *FloatHistogramChunk {
 	return &FloatHistogramChunk{b: bstream{stream: b, count: 0}}
 }
 
+// NewPrepopulatedFloatHistogramChunk returns the decoded version of a previously-encoded chunk.
```

Comment:
I don't quite understand this doc comment. Is "decoded" really the right word? Isn't it more along the lines "... returns a previously encoded chunk, with its internal bstream set up to accept new samples."

Reply:
Ah, yea decoded is a confusing choice. I was trying to come up with "the opposite of encoded" I think. I like your wording--I'll go with that.